src/containers/BooksForm.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { addBook } from '../actions/index';
+
+
+const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      category: 'default',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const newState = event.target.id === 'title' ? { title: event.target.value } : { category: event.target.value };
+
+    this.setState(newState);
```

Comment:
This documentation could be useful for this: https://reactjs.org/docs/forms.html#handling-multiple-inputs